state/compatibility_test.go

```diff
@@ -98,81 +140,157 @@ func TestSaveLegacyAndLoadFinalizeBlock(t *testing.T) {
 	multiStore := NewMultiStore(stateDB, options, store)
 
 	// try with a complete ABCI Response
-	abciResponses := newV1Beta2ABCIResponses()
-	require.Equal(t, 1, len(abciResponses.DeliverTxs))
-	require.Equal(t, 1, len(abciResponses.BeginBlock.Events))
-	require.Equal(t, 1, len(abciResponses.EndBlock.Events))
```

Comment:
Could you keep these three tests? This way we make sure that e.g. check in line 157 isn't passing because 0 == 0... in this case line 158 will panic

Reply:
good point, I've added the tests